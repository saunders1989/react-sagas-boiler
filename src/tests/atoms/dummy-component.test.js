import React from 'react';
import { shallow } from 'enzyme';

import DummyComponent from '../../components/atoms/dummy-component';

/** @test {DummyComponent} */
describe('DummyComponent', () => {

  it('should be defined', () => {
    expect(DummyComponent).toBeDefined();
  });

  let wrapper;

  it('should render correctly', () => {
    wrapper = shallow(
      <DummyComponent />
    );
    expect(wrapper).toMatchSnapshot();
  });
});
