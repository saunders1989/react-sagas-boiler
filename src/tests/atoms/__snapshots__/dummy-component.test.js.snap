// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DummyComponent should render correctly 1`] = `
<button
  onClick={[Function]}
  type="button"
>
  Click to increment
</button>
`;
